<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="formGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры модели</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество требований</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество устройств</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Емкость очереди</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="formGroupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры генераторов</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>SEED A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>SEED S</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>SEED P</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="formGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Приоритеты</string>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>451</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mplvl">
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="verticalGroupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Графики</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>Коэффициент использования системы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Число требований в очереди</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Число требований в системе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Среднее число требований в очереди</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Среднее число требований в системе</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="formGroupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Характеристики</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="horizontalSpacing">
      <number>25</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Tq</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Ts</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Nq</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Ns</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Ca</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Cr</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label0">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label6">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="formGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>160</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Число прогонов</string>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
